### ISSUE TITLE
Deprecation: Upgrade ember-cli-babel 5.x

### ISSUE THREAD

**Issue by belgoros** [lozjackson/ember-time-tools#40]
When installing the add-on, I got the wollowing warning:
```
EPRECATION: ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6. Version 5.2.8 located: ....
```
Are you going to upgrade `ember-cli-babel` ?

I'm using
```
ember-cli: 3.0.0
node: 8.9.1
os: darwin x64
```
Thank you.

**Comment by lozjackson** [comment]
@belgoros yes, I will upgrade it possibly later today, or early next week.. (if you don't want to wait PRs are welcome)

**Comment by lozjackson** [comment]
I've updated the addon to ember-cli 2.18.2 in PR #42 and released as [`v1.0.0`](https://github.com/lozjackson/ember-time-tools/releases/tag/v1.0.0)
